### PR TITLE
ci: build and push linux docker image to aws

### PR DESCRIPTION
**Pull request by Agnar22**

### CODE REVIEW

File: .github/workflows/linux_release.yml [hunk 1]
```diff
@@ -75,6 +77,30 @@ jobs:
...
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+        if: startsWith(github.ref, 'refs/tags/')
+
+      - name: Build docker image
+        working-directory: ${{ runner.temp }}/shadow_build_dir
+        if: startsWith(github.ref, 'refs/tags/')
+        run:  |
+              docker build --build-arg APPIMAGE_PATH=${{ runner.temp }}/shadow_build_dir/package/${{ env.ARTIFACT }} ${SOURCE_DIR}/.github/workflows/Dockerfile -t ${ECR_REGISTRY}/${ECR_REPOSITORY}:latest ${ECR_REGISTRY/${ECR_REPOSITORY}:${{ github.ref }}
```
Comment (kibidev) [inline .github/workflows/linux_release.yml:97]: I think "github.ref" translate to "refs/tags/v4.2.0-1.8.1" and similar, better to use "github.ref_name" whouch resolves to the tag name without the "refs/tags/" prefix

**Review by kibidev — CHANGES_REQUESTED** [review]
Tip (no need to change here) is to prefix branch name with "aviant/" to trigger test build in PR.
Otherwise looks good, just two minor requests.

File: .github/workflows/Dockerfile
[the remark this answers is not in the document]
Comment (Agnar22) [reply .github/workflows/Dockerfile:1]: Fixed.

File: .github/workflows/linux_release.yml (hunk 1 above)
Comment (Agnar22) [reply .github/workflows/linux_release.yml:97]: Fixed.

**Comment by Agnar22** [comment]
Pushing to ECR is tested and works!

**Review by kibidev — APPROVED** [review]

**Pull request merged**